src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts

```diff
@@ -248,6 +248,7 @@ export class HaVoiceCommandDialog extends LitElement {
       css`
         ha-dialog {
           --mdc-dialog-max-width: 500px;
+          --mdc-dialog-min-width: var(--mdc-dialog-max-width);
```

Comment:
This change breaks mobile. You need to set it for mobile too:

`@media all and (max-width: 450px), all and (max-height: 500px) {`

Reply:
Good catch. Reverting that change as I don't want to keep all those changes in sync.